container/root/goss.nginx.yaml

```diff
@@ -1,3 +1,5 @@
+gossfile:
+  goss.base.yaml: {}
```

Comment:
I like that we can rerun the parent container tests again in downstream containers. 🔨